app/assets/stylesheets/components/buttons.scss

```diff
@@ -48,7 +48,8 @@
 
   &[href]:hover,
   &:hover:enabled,
-  &:active:enabled {
+  &:active:enabled,
+  &:focus:enabled {
```

Comment:
I've decided to also the focus effect to the `next-button` which is using `.crayons-btn`. This change should be harmless but would love to hear input from @ludwiczakpawel

Reply:
Fascinating. Looks cool.